apps/web/src/hooks/workflowChannelPreferences/useUpdateWorkflowChannelPreferences.ts

```diff
@@ -17,6 +18,7 @@ export const useUpdateWorkflowChannelPreferences = (
     WorkflowChannelPreferences
   >((data) => api.upsertPreferences(workflowId, data), {
     onSuccess: () => {
+      onSuccess();
       successMessage('Workflow Channel Preferences updated');
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** why are we having a separate success message for preferences? When clicking save button it updates all changes - both of preferences and changes in the general tab. 
Follow up: when saving after changes only in the general tab there is no feedback to the user - no success message - only changing the save button to be disabled again ‏

Reply:
I did not implement that part originally, but agree, I should have removed the message once I combined things into a single "save" button. I'll do it now!